static/src/javascripts/lib/config.js

```diff
@@ -3,7 +3,15 @@
 // This should be the only module accessing the window config object directly
 // because this is the one that gets imported to all other modules
 // eslint-disable-next-line guardian-frontend/global-config
-const config = window.guardian.config;
+const config: Object = window.guardian.config;
+
+// allows you to safely get items from config using a query of
+// dot or bracket notation, with optional default fallback
+const get = (path: string = '', defaultValue: any) =>
```

Comment:
Minor detail here: `get(path, defaultVal)` is less secure/efficient than `get(path) || defaultVal`, as `defaultVal` is not evaled in the latter case.